hardhat-project/contracts/SVG.sol

```diff
@@ -1,6 +1,8 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.2;
 
+import "hardhat/console.sol";
+
```

Comment:
I don't think this import is affecting the contract size unless I use the `console` library functions.